ctru-sys/bindgen.sh

```diff
@@ -29,3 +28,5 @@ bindgen "$DEVKITPRO/libctru/include/3ds.h" \
     -DARM11 \
     -D__3DS__ \
 > src/bindings.rs
+
+cargo run --bin docstring-to-rustdoc -- src/bindings.rs
```

Comment:
Related (fyi): https://github.com/rust-lang/rust-bindgen/issues/955

Reply:
Ohh, I didn't see that until now! Thanks, I'll check it out!

Reply:
Nothing's implemented there, but someone else did have the idea to open an issue at least.